language/German/strings.po

```diff
@@ -1138,16 +1159,15 @@ msgid "Emulated"
 msgstr "Emuliert"
 
 #: src/frontend/mame/ui/keyboard.cpp:60 src/frontend/mame/ui/confswitch.cpp:117
-#: src/frontend/mame/ui/inputmap.cpp:578
-#: src/frontend/mame/ui/inputtoggle.cpp:73
+#: src/frontend/mame/ui/inputmap.cpp:578 src/frontend/mame/ui/inputtoggle.cpp:73
 #: src/frontend/mame/ui/analogipt.cpp:361
```

Comment:
Please back out the re-wrapped source locations.

Reply:
Fixed